pkg/internal/controller/controller.go

```diff
@@ -99,7 +105,6 @@ type watchDescription struct {
 
 // Reconcile implements reconcile.Reconciler.
 func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (_ reconcile.Result, err error) {
-	log := c.Log.WithValues("name", req.Name, "namespace", req.Namespace)
```

Comment:
Can we leave this? It is somewhat redundant, but it will keep existing queries working

Reply:
I'm not sure if we want to have the redundancy going forward / forever.

WDYT about the following options:
* keeping it for now, but deprecating it (I'm not sure how) and then drop it with a later minor release (not the next one)
* providing a LogConstructor which uses either the old format or the new one + this k/v pair. In this case folks can still use the old format but would have to set that constructor explicitly

Reply:
Apart from having it twice in the log which makes it slightly less readable, I assume there are no drawbacks.

I assume systems like Elastic and Loki are optimized enough that this doesn't really require more storage.

Then I'll keep it.

Reply:
I added it in the log constructors to keep it close to where I already add the object. If someone has strong opinions on that they can pass in their own LogConstructor.

PTAL.